civiform/civiform
# [Program Card Images] Have image upload be the second step in the program creation flow

### Description

We want to encourage admins to upload images for all their programs. To help with this, this PR adds image upload as a second step in the program creation flow.

Previously, once an admin creates a new program and fills in the details, they'd be taken to the main program edit page. With this PR, once the admin creates a new program, they'll be taken to the image upload page instead.

When an admin chooses "Edit program details" on an existing draft program, that page will *also* redirect to the image upload page once the details have been saved.

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### New Features


### Instructions for manual testing

1. Enable the PROGRAM_CARD_IMAGES flag.
2. On the program dashboard, click "Create new program".
3. Fill in some program details.
4. Verify the button on the bottom of the page says "Save and continue to next step".
5. Click that button.
6. Verify that you're taken to the image upload page.


### Issue(s) this completes

Epic #5676 

Fixes #6000
